magefiles/magefile.go

```diff
@@ -93,7 +93,13 @@ func (CI) parseJobSpec() error {
 func (ci CI) init() error {
 	var err error
 
-	if jobType == "periodic" || strings.Contains(jobName, "rehearse") {
+	/*
+		Do not parse CI jobs data when:
+		 1. Openshift CI periodic jobs
+		 2. When testing in openshift/release repo
+		 3. When a integration test is coming from Konflux with a push event_type
+	*/
+	if jobType == "periodic" || strings.Contains(jobName, "rehearse") || konfluxCiSpec.KonfluxGitRefs.EventType == "push" {
```

Comment:
if we don't initialize the config for konflux periodic jobs, we won't be able to identify for which component the periodic job runs for, no? so by default, all tests suites will run for build-service, integration-service etc...